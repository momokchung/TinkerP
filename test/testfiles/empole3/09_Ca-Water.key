parameters       amoeba09

digits 10
