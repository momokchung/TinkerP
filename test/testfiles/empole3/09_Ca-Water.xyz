     4  Water and Calcium Ion
     1  O      1.508930    0.000000    0.000000    39     2     3
     2  H      2.132040    0.757179    0.000000    40     1
     3  H      2.132040   -0.757179    0.000000    40     1
     4  Ca2   -0.709606    0.000000    0.000000    13
