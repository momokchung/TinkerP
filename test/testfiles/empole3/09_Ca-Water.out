
     ######################################################################
   ##########################################################################
  ###                                                                      ###
 ###            Tinker  ---  Software Tools for Molecular Design            ###
 ##                                                                          ##
 ##                      Version 8.10.5   February 2023                      ##
 ##                                                                          ##
 ##               Copyright (c)  Jay William Ponder  1990-2023               ##
 ###                           All Rights Reserved                          ###
  ###                                                                      ###
   ##########################################################################
     ######################################################################


 Intermolecular Energy :             -57.10095213 Kcal/mole

 Total Potential Energy :            -55.62844560 Kcal/mole

 Energy Component Breakdown :           Kcal/mole        Interactions

 Bond Stretching                       0.57493335                2
 Angle Bending                         0.90041644                1
 Urey-Bradley                         -0.00284326                1
 Van der Waals                        16.90340512                3
 Atomic Multipoles                   -48.51887287                3
 Polarization                        -25.48548438                3
